packages/stitch/src/mergeCandidates.ts

```diff
@@ -285,7 +377,10 @@ function mergeScalarTypeCandidates(
   const astNodes = pluck<ScalarTypeDefinitionNode>('astNode', candidates);
   const astNode = astNodes
     .slice(1)
-    .reduce((acc, astNode) => mergeScalar(acc, astNode), astNodes[0]) as ScalarTypeDefinitionNode;
```

Comment:
Update to match the other mergers. This one was setup slightly differently and was producing different results.